src/EditorFeatures/Core/Implementation/RenameTracking/AbstractRenameTrackingCodeFixProvider.cs

```diff
@@ -43,11 +43,8 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
             // any fixes.
             if (RenameTrackingTaggerProvider.CanInvokeRename(document))
             {
-                var action1 = RenameTrackingTaggerProvider.CreateCodeAction(document, diagnostic, _waitIndicator, _refactorNotifyServices, _undoHistoryRegistry, showPreview: false);
+                var action1 = RenameTrackingTaggerProvider.CreateCodeAction(document, diagnostic, _waitIndicator, _refactorNotifyServices, _undoHistoryRegistry);
```

Comment:
I'm pretty sure I had renamed `action1` to `action`, but seems like I have lost that change. :fearful: The feature as such works in my testing. Will fix this.